data/coalition/lunarium intro.txt

```diff
@@ -1285,20 +1285,20 @@ mission "Lunarium: Evacuation 6"
+			`	"Elirom, I am. Born on Saros. Raised there, fascinated by the Heliarchs since my infancy. When of age I came, into the Academy of Cadets I enlisted. Served the Heliarchy for 67 years, I had, before finally the right thing, I did. For the last 18 of those years, an arbiter, I was." He brings his hands together, as if to warm them up. "The Lunarium Redoubt, I then joined. Belong to that group, they do, the people whom rescue, you did. Or, at least, their parents do... did." He sighs. "Criminals, outlaws... call us whatever names the Heliarchs do, simply trying to save our Coalition from them, we are. Saving people from their mock hunts for terrorists, my job is. Believe me, you may not. Choose to see us, you can, as what deem us, the Heliarchs do. In doing that, though, changing the things they do, the things that seen, I have, the things that..." he trails off. "But dead to them for decades now, I have been."`
 			choice
 				`	"You want me to join your group in fighting the Heliarchs, is that it?"`
 					goto join
 				`	"Dead? What do you mean?"`
-			`	He thinks for a few minutes, oftentimes opening up his mouth as if deciding to speak, but ultimately going back to reflecting on what to say. "When decided to... desert, I had, work I did on covering up my fleeing of the facility I was stationed at. After forcing an... an incident, meant to erase all traces of mine, fled and hid in civilian ships for a while, I had, until found by the Lunarium I was." He smirks lightly, as if in bitterness. "Thought I did that aware enough of their patterns and method, I was, enough to locate them. Found me first, they did though. Taken in, I was, and helping them since, I have been."`
+			`	He thinks for a few minutes, oftentimes opening up his mouth as if deciding to speak, but ultimately going back to reflecting on what to say. "When decided to... desert, I had, work I did on covering up my escape of the facility where stationed, I was. After forcing an... an incident, meant to erase all traces of mine, fled and hid in civilian ships for a while, I had, until found by the Lunarium I was." He smirks lightly in self-deprecation. "Thought I had that aware enough of their patterns and methods, I was, enough to locate them. Find me first, they did though. Taken in, I was, and helping them since, I have been."`
```

Comment:
Why change bitterness to self-deprecation?

Reply:
Does the placement there necessarily mean the bitterness is about that? The idea I had for this was while he says all of this he's thinking about more than just how it was easier for the Lunarium to find him than the other way around. Him being bitter would be about him reflecting on not joining them earlier, stuff like that, but as we don't go and peer into characters' thoughts it couldn't be stated.

If that can't really translate without it implying the bitterness is about the fact of him being found by the Lunarium before he could find them, I'd rather just cut the sentence at "He smirks lightly" as self-deprecation doesn't seem appropriate to me either, as that's not what he's feeling.

Reply:
Yeah, just cutting it at "lightly" seems best.